sbin/common/common.sh

```diff
@@ -173,7 +173,30 @@ function setBootJdk() {
     fi
 
     echo "Guessing JDK_BOOT_DIR: ${BUILD_CONFIG[JDK_BOOT_DIR]}"
-    echo "If this is incorrect explicitly configure JDK_BOOT_DIR"
+    echo "=BOOT JDK VERSION OUTPUT="
+    ${BUILD_CONFIG[JDK_BOOT_DIR]}/bin/java -version
```

Comment:
I think we already show this somewhere else so it would be best not to duplicate this